src/e-load-xcode.js

```diff
@@ -48,6 +53,29 @@ for (const sdk of SDK_TO_LINK) {
   childProcess.execFileSync('sudo', ['ln', '-s', sourceSDK, targetDirectory]);
 }
 
+// Unlink unnecessary macOS SDKs.
+for (const sdk of SDK_TO_UNLINK) {
+  // Check that source SDK exists.
+  const sourceSDK = path.resolve(macOSSDKs.path, `MacOSX${sdk}.sdk`);
+  if (!fs.existsSync(sourceSDK)) continue;
+
+  // Check that target exists.
+  const targetDirectory = path.resolve(xCodeSDKDir, `MacOSX${sdk}.sdk`);
+  if (!fs.existsSync(targetDirectory)) continue;
+
+  // Check that target is a valid symbolic link.
+  const stats = fs.lstatSync(targetDirectory);
+  if (!stats.isSymbolicLink()) return;
+
+  console.warn(
+    `${color.info} Removing symbolic link from ${color.path(sourceSDK)} --> ${color.path(
+      targetDirectory,
+    )}`,
+  );
+
+  childProcess.execFileSync('unlink', [sourceSDK]);
```

Comment:
Shouldn't this be `unlink', [targetDirectory])` ?

Reply:
derp, ye. fixed.